File: php/Processor.php
<?php 
 
 class Processor{

    //atribut
	private $name;
	private $price;
		
	public function __construct(){
	}
		
		
	public function setName($name){
		$this->name = $name;
	}
		
	public function getName(){
		return $this->name;
	}
		
	public function setPrice($price){
		$this->price = $price;
	}
		
	public function getPrice(){
		return $this->price;
	}
	public function	printProcessor(){
        echo "Processor~ <br>";
        echo "Name  : ".$this->name. "<br>";
        echo "Price : Rp ".$this->price. "<br>";
    }
	function __destruct(){
	}
}
?>

		
		

File: python/Processor.py
class Processor():
    #private members
    __name = ""
    __price = 0

    def __init__(self, name = "", price = 0):
        self.__name = name 
        self.__price= price

    #setter getter
    def setName(self, name):
        self.__name = name 
    
    def getName(self):
        return self.__name

    def setPrice(self, price):
        self.__price = price 
    
    def getPrice(self):
        return self.__price
    
    def printProcessor(self):
        print("Processor~ ")
        print("Name  : %s" % (self.__name))
        print("Price : Rp %d" %(self.__price))